source/documentation/using_the_api/api_reference.md

```diff
@@ -10,48 +10,45 @@ View information about a register, including:
 * when the register was last updated
 * how many entries and records the register contains
 
-Example URL: https://country.register.gov.uk/register
+Example URL: `https://local-authority-eng.register.gov.uk/register/`
 
-Example request: `curl --request GET --url https://country.register.gov.uk/register --header 'accept: application/json'`
+Example request: `curl --request GET --url https://local-authority-eng.register.gov.uk/register/ --header 'accept: application/json'`
 
 Example response:
 
 ```
-{  
-   "domain":"register.gov.uk",
-   "total-records":199,
-   "total-entries":206,
-   "register-record":{  
-      "fields":[  
-         "country",
-         "name",
-         "official-name",
-         "citizen-names",
-         "start-date",
-         "end-date"
-      ],
-      "registry":"foreign-commonwealth-office",
-      "text":"British English-language names and descriptive terms for countries",
-      "phase":"beta",
-      "register":"country",
-      "entry-timestamp":"2016-08-04T14:45:41Z",
-      "key":"country",
-      "index-entry-number":"2",
-      "entry-number":"2"
-   },
-   "last-updated":"2017-03-29T14:22:30Z"
+{
+  "domain": "register.gov.uk",
+  "total-records": 354,
+  "total-entries": 358,
+  "register-record": {
+    "fields": [
+      "local-authority-eng",
+      "local-authority-type",
+      "name",
+      "official-name",
+      "start-date",
+      "end-date"
+    ],
+    "registry": "department-for-communities-and-local-government",
+    "text": "Local authorities in England",
+    "phase": "beta",
+    "register": "local-authority-eng"
+  },
+  "custodian": "Mark Coram",
+  "last-updated": "2017-07-04T10:55:43Z"
 }
 ```
 
 ### Get all records
 
 Path: GET /records
 
-Get all records from the register. For example, all of the countries from the country register.
+Get all records from the register. For example, all of the English local authorities from the 'Local authority eng' register.
```

Comment:
This casing is a bit weird. As this is an API reference. I would rather use the actual register name i.e. `local-authority-eng`.